cypress/e2e/awx/resources/inventorySource.cy.ts

```diff
@@ -70,7 +77,17 @@ describe('Inventory source page', () => {
     cy.selectDropdownOptionByResourceName('source_control_type', 'Sourced from a Project');
     cy.selectDropdownOptionByResourceName('project', project.name);
     cy.selectDropdownOptionByResourceName('inventory', 'Dockerfile');
-    cy.getBy('.pf-v5-c-input-group > .pf-v5-c-button').click();
+    cy.getBy('[data-cy="execution-environment-select-form-group"]').within(() => {
+      cy.getBy('.pf-v5-c-input-group > .pf-v5-c-button').click();
+    });
+    cy.getBy('[data-ouia-component-type="PF5/ModalContent"]').within(() => {
```

Comment:
Lets try to find a different locator.  We avoid using these locators, because they come from patternfly, and they are subject to change, which would break out tests.

Reply:
That makes sense, I can poke around and see if there's a better option.

Reply:
It looks like we are setting `data-ouia-component-id` explicitly so I updated these to use that instead.